include/nbl/asset/filters/CBlitImageFilter.h

```diff
@@ -189,21 +267,43 @@ class CBlitImageFilter : public CImageFilter<CBlitImageFilter<Swizzle,Dither,Nor
 					};
 				};
 				
-				uint32_t				inMipLevel = 0u;
-				uint32_t				outMipLevel = 0u;
-				ICPUImage*				inImage = nullptr;
-				ICPUImage*				outImage = nullptr;
-				KernelX					kernelX;
-				KernelY					kernelY;
-				KernelZ					kernelZ;
+				uint32_t							inMipLevel = 0u;
+				uint32_t							outMipLevel = 0u;
+				ICPUImage*							inImage = nullptr;
+				ICPUImage*							outImage = nullptr;
+				KernelX								kernelX;
+				KernelY								kernelY;
+				KernelZ								kernelZ;
+				bool								enableLUTUsage = true; // temporary state for testing, will remove before merge
 		};
 		using state_type = CState;
 		
+		// If you're changing this method don't forget to change the one below (getPhaseSupportLUTByteOffset)
```

Comment:
DRY your code, then you won't need this comment

```cpp
static inline uint32_t getRequiredScratchByteSize(const state_type* state)
{
    uint32_t retval = getPhaseSupportLUTByteOffset(state);
    // need to add the memory for phase support LUT
			const core::vectorSIMDu32 phaseCount = getPhaseCount(state->inExtentLayerCount, state->outExtentLayerCount, state->inImage->getCreationParameters().type);
			retval += ((phaseCount[0]*scaledKernelX.getWindowSize().x) + (phaseCount[1]*scaledKernelY.getWindowSize().y) + (phaseCount[2]*scaledKernelZ.getWindowSize().z)) * sizeof(value_type);

			return retval;
}
```

Reply:
Oh yeah.. it was weird.

Resolved.